src/Datadog.Trace/Sampling/DefaultSamplingRule.cs

```diff
@@ -17,6 +17,8 @@ internal class DefaultSamplingRule : ISamplingRule
         /// </summary>
         public int Priority => int.MinValue;
 
+        internal static bool UseOptim { get; set; }
```

Comment:
I think you had this here for benchmarking and forgot to remove it.

Reply:
I checked and double-checked, and yet